src/casp/services/api/data_manager/cellarium_general.py

```diff
@@ -40,6 +40,18 @@ def get_application_info() -> schemas.ApplicationInfo:
             default_feature_schema=settings.DEFAULT_FEATURE_SCHEMA, application_version=settings.APP_VERSION
         )
 
+    def feedback_opt_out(self, user: models.User) -> None:
+        """
+        Opt out user from feedback requests
+
+        :param user: User object to opt out
+        """
+        user.feedback_opt_out = True
+        with self.system_data_db_session_maker() as session:
```

Comment:
If you use `with self.system_data_db_session_maker.begin() as session:`, it will create a transaction that will implicitly commit, so you don't need to explicitly call commit

Reply:
Although the way you're doing it now might be cleaner since you're also getting the updated row.  I'm not sure

Reply:
Yeah, with the fetch it seemed to do something to the commit.  I figured it'd be safer to do the write + read + commit explicitly in the session